com.unity.netcode.gameobjects/Tests/Runtime/NetworkAnimator/NetworkAnimatorTests.cs

```diff
@@ -18,10 +18,21 @@ public class NetworkAnimatorTests : BaseMultiInstanceTest
         private Animator m_PlayerOnServerAnimator;
         private Animator m_PlayerOnClientAnimator;
 
+
         [UnitySetUp]
         public override IEnumerator Setup()
         {
-            yield return StartSomeClientsAndServerWithPlayers(useHost: true, nbClients: NbClients,
+            // this is treacherous...normally BaseMultiInstance calls StartSomeClientsAndServerWithPlayers for you
```

Comment:
@NoelStephensUnity this was what I was trying to convey in our slack thread

Reply:
There is a way to handle this a little more gracefully in the NetcodeIntegraitonTest, and was why I included an example of how to do this in the documentation (because the way it is currently can be problematic to figure out the best way to achieve the same kind of results).